tailwind/app_template_v2/static_src/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "django_tailwind",
+  "description": "",
+  "scripts": {
+    "start": "npm run dev",
+    "build": "npm run build:clean && npm run build:sass && npm run build:postcss && npm run build:cleancss",
+    "build:clean": "rimraf ../static/css",
+    "build:sass": "node-sass --output-style compressed src/styles.scss ../static/css/styles.css",
+    "build:postcss": "postcss --config . --map false --output ../static/css/styles.css ../static/css/styles.css",
+    "build:cleancss": "cleancss -o ../static/css/styles.css ../static/css/styles.css",
+    "dev": "watch 'npm run dev:sass && npm run dev:postcss' ./src",
+    "dev:sass": "node-sass --output-style expanded --source-map true src/styles.scss ../static/css/styles.css",
+    "dev:postcss": "postcss --config . --map true --output ../static/css/styles.css ../static/css/styles.css"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "autoprefixer": "^10.0.2",
+    "clean-css-cli": "^4.3.0",
+    "node-sass": "^4.14.1",
+    "postcss": "^8.1.9",
```

Comment:
PostCSS is now at version 8.1.10. Obviously a moving target, but might as well update to the latest version since this hasn't been merged yet. 😁